docs/_snippets/patient-response-no-facilities-or-links.mdx

```diff
@@ -3,6 +3,11 @@
   documents.
 </ResponseField>
 
+<ResponseField name="externalId" type="string">
+  The ID assigned to this Patient by the Client. This ID will be used to uniquely identify this
+  Patient in the Client's system.
+</ResponseField>
```

Comment:
let's make this consistent with the other description - ie "An external Patient ID to associate to a Patient in Metriport."